### PR TITLE
FOUR-19364: When a task is open show the component Case Comments

### PR DESCRIPTION
**Pull request by henryjonathanquispe**
## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-19364

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:deploy

### CODE REVIEW

**Review by pmPaulis — APPROVED** [review]

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-64c17b607d.engk8s.processmaker.net

File: resources/views/tasks/edit.blade.php [hunk 1]
```diff
@@ -868,6 +869,16 @@ class="mr-2 custom-badges pl-2 pr-2 rounded-lg">
           },
           onInput(filter) {
             this.getUsers(filter);
+          },
+          getCommentsData: async () => {
```
Comment (fagubla) [inline resources/views/tasks/edit.blade.php:873]: missing catch error

**Review by fagubla — CHANGES_REQUESTED** [review]

File: resources/views/tasks/edit.blade.php (hunk 1 above)
Comment (henryjonathanquispe) [reply resources/views/tasks/edit.blade.php:873]: The commentContainer component has the success handler for this method, for else it does nothing.

**Review by boliviacoca — APPROVED** [review]

**Comment by pmPaulis** [comment]
@henryjonathanquispe 
Currenlty from task in not showing the subject
current
image
expected
image

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-64c17b607d.engk8s.processmaker.net

**Review by pmPaulis — APPROVED** [review]

**Review by fagubla — APPROVED** [review]

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-64c17b607d.engk8s.processmaker.net

**Pull request merged**